### ISSUE TITLE
ImportError running predict.py

### ISSUE THREAD

**Issue by cpudney** [milesial/Pytorch-UNet#284]
Following the package reorganisation I get the following error from `predict.py`

```
$ python predict.py
Traceback (most recent call last):
  File "predict.py", line 13, in <module>
    from utils import plot_img_and_mask
ImportError: cannot import name 'plot_img_and_mask' from 'utils' (/home/cp/PycharmProjects/Pytorch-UNet/utils/__init__.py)
```

Thanks,
Chris.

**Comment by Litou-lyh** [comment]
you can change "from utils import plot_img_and_mask" to "from utils.utils import plot_img_and_mask"

**Comment by milesial** [comment]
Ah my bad! Thanks for opening the bug, this is now fixed with faa6e13

**Comment by cpudney** [comment]
Yep - can confirm it's fixed now.
